src/targets/gpu/kernels/include/migraphx/kernels/hip.hpp

```diff
@@ -29,9 +29,8 @@
 #include <hip/hip_fp16.h>
 #include <hip/math_functions.h>
 #include <hip/hip_math_constants.h>
-#elif defined(MIGRAPHX_ENABLE_HIPRTC_WORKAROUNDS)
-#include <hip/hip_common.h>
-#include <hip/hip_math_constants.h>
+#else
+constexpr auto HIP_PIO2_F = 1.570796327F;
```

Comment:
This should use a different name so it doesnt conflict with hip's macro, and it should go in the math.hpp header probably.

Reply:
Removed workaround for `sin` instead